projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/vdkTags.ts

```diff
@@ -98,17 +112,23 @@ export const trackVdkTags = (
           vdkCellIndices.push(cellIndex);
         cellIndex++;
       }
-
+      // this case covers the use case when the notebook is loaded for the first time
+      if (!vdkCellIndices.length) {
+        vdkCellIndices = await getVdkCellIndices(
+          notebookTracker.currentWidget.context.path
+        );
+      }
       if (
+        vdkCellIndices.length > 0 &&
         notebookTracker.activeCell &&
         notebookTracker.activeCell.parent &&
         notebookTracker.activeCell.parent.node.children
       ) {
         addVdkCellDesign(
-          notebookTracker.activeCell.node,
           Array.from(notebookTracker.activeCell.parent.node.children!),
```

Comment:
You can simplify this with the elvis operator, then you won't need any checks. Something like

```
Array.from(notebookTracker?.activeCell?.parent?.node?.children)
```

Not sure about warnings, though, you might have to experiment a bit.

Reply:
@DeltaMichael I get codify warnings bc of them, I used to write them like that and stopped bc of it 
Should I go back to them?

Reply:
Can you check the behaviour if you use elvis and something is missing? Like if activeCell.parent is undefined, for some reason. Sorry, I'm not entirely sure what happens in this case either when you pass it to `Array.from()`. 😕

Reply:
@DeltaMichael in this case nothing happens, the cells remain in their default Jupyter design but I still get warnings, I will keep it like that for now